Makefile

```
@@ -694,15 +683,11 @@ clean-image:
 
 .PHONY: tag
 tag:
-ifneq (,$(wildcard CI_TAG))
-	@cat CI_TAG
-else
 ifdef COMMIT
 	@git describe $(COMMIT) --tags --abbrev=10 --long --exclude '*-nightly-*'
 else
 	@echo $(TAG)
 endif
```

WDYT about dropping this if-branch with `COMMIT` altogether? It seems to me was created for convenience of getting a `tag` for an arbitrary commit. I'm not sure anyone uses it and I worry that the command used here may drift compared to the one used for constructing `$(TAG)`.
https://github.com/stackrox/rox/pull/1173

```suggestion
	@echo $(TAG)
```

---

+1 to drop if no one is using it but not in this PR.